ccache.c

```diff
@@ -2329,21 +2331,31 @@ cc_process_args(struct args *args, struct args **preprocessor_args,
 		/* Debugging is handled specially, so that we know if we can strip line
 		 * number info. */
 		if (str_startswith(argv[i], "-g")) {
-			generating_debuginfo = true;
-			args_add(stripped_args, argv[i]);
-			if (conf->unify && !str_eq(argv[i], "-g0")) {
-				cc_log("%s used; disabling unify mode", argv[i]);
-				conf->unify = false;
-			}
-			if (str_eq(argv[i], "-g3")) {
-				/*
-				 * Fix for bug 7190 ("commandline macros (-D)
-				 * have non-zero lineno when using -g3").
-				 */
-				cc_log("%s used; not compiling preprocessed code", argv[i]);
-				conf->run_second_cpp = true;
+			const char *pLevel = argv[i] + 2;
+			int foundlevel = -1;
+			if (str_startswith(argv[i], "-ggdb"))
```

Comment:
Please add curly braces around `if` bodies even if it only contains one statement.